Readme.md

```diff
@@ -386,6 +388,174 @@ export function MyForm({ isNewItem, addItem, updateItem }: Props): JSX.Element =
 
 ---
 
+## Nested Objects
+
+Each FormField object is also able to contain fields. It behaves similar as with the Form object but with slight differences on it's methods (See [API](#nested-fields)).
+
+```ts
+import { useForm } from 'react-use-form-library';
+
+function App(): JSX.Element {
+  const { model, fields, onSubmit } = useForm({
+    model: {
+      address: {
+        streetName: '',
+      },
+    },
+    handleSubmit: async () => {
+      console.log(model);
+    },
+  });
+
+  return (
+    <form onSubmit={onSubmit}>
+      <input
+        /* Since the FormField could be either a value or an object, the fields property is optional 
```

Comment:
I don't really get this restriction. From the type system it should be clear at this point that this field is a required object.

Reply:
You're right. This will indeed give you a typescript error based on your model type.